sklearn/semi_supervised/tests/test_label_propagation.py

```diff
@@ -59,3 +62,15 @@ def test_predict_proba():
         clf = estimator(**parameters).fit(samples, labels)
         assert_array_almost_equal(clf.predict_proba([[1., 1.]]),
                                   np.array([[0.5, 0.5]]))
+
+
+def test_alpha_deprecation():
+    X, y = make_classification(n_samples=100)
+    y[::3] = -1
+    lp_default = label_propagation.LabelPropagation()
+    lp_default_y = assert_no_warnings(lp_default.fit, X, y).transduction_
```

Comment:
I don't seem to get that warning on my system, but all Travis runs get it.

Could I suggest you try setting the `kernel='knn'` in the test here, and we'll see if that works?